tsconfig.base.json

```diff
@@ -11,7 +11,7 @@
     "esModuleInterop": true,
     "target": "es2015",
     "module": "esnext",
-    "lib": ["es2020", "dom"],
+    "lib": ["es2022", "dom"],
```

Comment:
Is this for `Object.hasOwn()`? caniuse.com looks good from a support perspective, but can you confirm with Sean that this aligned with the Grid's expectations for minimum browser API level we support?

Reply:
Sure, I'll ask him about it... (yes, that's mainly for `Object.hasOwn`